website_livechat_firstname/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Website Live Chat - First Name
+==============
+
+This module shows only the first name of an employee in Live Chat.
+
+Installation
```

Comment:
Probably no need for this section, seeing as the installation instructions are simply "Install module"